src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
+// not my solution
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averGrade'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order:SortOrder,
+): Student[] {
+  const sorted: Student[] = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      return order === 'asc'
+        ? sorted.sort((a: Student, b: Student) => a[sortBy]
```

Comment:
You don't need to specify types here, cause typescript knows you sort `Student[]`
```suggestion
        ? sorted.sort((stundentA, stundentB) => a[sortBy]
```